### PR TITLE
Fix GA landing page

### PR DESCRIPTION
**Pull request by rhelmer**
# References: 
Jira: MNTOR-
Figma: 

# Description

# Screenshot (if applicable)

Not applicable.

# How to test

# Checklist (Definition of Done)
- [ ] Localization strings (if needed) have been added.
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: src/app/(proper_react)/(redesign)/(public)/LandingView.tsx [hunk 1]
```diff
@@ -31,231 +31,236 @@ import { FaqSection } from "./Faq";
...
-      <header className={styles.hero}>
-        <div className={styles.heroContent}>
-          <h1>{props.l10n.getString("landing-all-hero-title")}</h1>
-          <p>
-            {props.l10n.getString(
-              props.eligibleForPremium
-                ? "landing-premium-hero-lead"
-                : "landing-all-hero-lead",
+    <>
+      <GaScript nonce={props.nonce} />
```
Comment (flozia) [inline src/app/(proper_react)/(redesign)/(public)/LandingView.tsx:48]: We don’t need `GaScript` anymore if we use the `GoogleAnalytics` component from Next.js in `layout.tsx`.
```suggestion
```

File: src/app/(proper_react)/(redesign)/(public)/LandingView.tsx [hunk 2]
```diff
@@ -31,231 +31,236 @@ import { FaqSection } from "./Faq";
 import { SignInButton } from "../../../components/client/SignInButton";
 import { RebrandAnnouncement } from "./RebrandAnnouncement";
 import { FeatureFlagName } from "../../../../db/tables/featureFlags";
+import { GaScript } from "../GaScript";
```
Comment (flozia) [inline src/app/(proper_react)/(redesign)/(public)/LandingView.tsx:34]:
```suggestion
```

File: src/app/(proper_react)/(redesign)/(public)/LandingView.tsx [hunk 3]
```diff
@@ -31,231 +31,236 @@ import { FaqSection } from "./Faq";
...
 
 export type Props = {
   eligibleForPremium: boolean;
   l10n: ExtendedReactLocalization;
   countryCode: string;
   scanLimitReached: boolean;
   enabledFlags: FeatureFlagName[];
+  nonce: string;
```
Comment (flozia) [inline src/app/(proper_react)/(redesign)/(public)/LandingView.tsx:42]:
```suggestion
```

File: src/app/(proper_react)/(redesign)/(public)/page.tsx [hunk 4]
```diff
@@ -32,13 +32,16 @@ export default async function Page() {
   const scanLimitReached =
     typeof oneRepActivations === "undefined" ||
     oneRepActivations > monthlySubscribersQuota;
+
+  const nonce = headers().get("x-nonce") ?? "";
```
Comment (flozia) [inline src/app/(proper_react)/(redesign)/(public)/page.tsx:36]:
```suggestion
```

File: src/app/(proper_react)/(redesign)/(public)/page.tsx [hunk 5]
```diff
@@ -32,13 +32,16 @@ export default async function Page() {
...
   return (
     <View
       eligibleForPremium={eligibleForPremium}
       l10n={getL10n()}
       countryCode={countryCode}
       scanLimitReached={scanLimitReached}
       enabledFlags={enabledFlags}
+      nonce={nonce}
```
Comment (flozia) [inline src/app/(proper_react)/(redesign)/(public)/page.tsx:44]:
```suggestion
```

File: src/app/layout.tsx [hunk 6]
```diff
@@ -14,8 +14,9 @@ import { SessionProvider } from "../contextProviders/session";
 import { authOptions } from "./api/utils/auth";
 import { metropolis } from "./fonts/Metropolis/metropolis";
 import { CONST_GA4_MEASUREMENT_ID } from "../constants";
-import { GoogleAnalytics } from "@next/third-parties/google";
 import { headers } from "next/headers";
+import { GoogleAnalyticsWorkaround } from "./components/client/GoogleAnalyticsWorkaround";
+import { getNonce } from "./deprecated/functions/server/getNonce";
```
Comment (flozia) [inline src/app/layout.tsx:19]: Should we use `const nonce = headers().get("x-nonce") ?? ""` to get `nonce` like we do [here](https://github.com/mozilla/blurts-server/pull/4161/files#diff-521160afb661a91142c95654d4bf5644f90993f401587bf71f4b5871e26662e6R36)?

**Comment by flozia** [comment]
@rhelmer Create https://github.com/mozilla/blurts-server/pull/4162 that would introduce the suggested changes in this PR.

**Comment by rhelmer** [comment]
Sorry, I forgot that we load the component from `layout` - I don't think we need this at all then.

**Pull request closed without merging**